### PR TITLE
This feature allows to inject imported css.

### PR DESCRIPTION
**Pull request by bokzor**
## I'm submitting a...

- [ ] Bug Fix
- [x] Feature
- [ ] Other (Refactoring, Added tests, Documentation, ...)

## Checklist

- [x] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern, see current [`.vcmrc`](https://github.com/dherges/ng-packagr/blob/master/.vcmrc)
- [ ] Tests for the changes have been added

## Description

_please describe the changes that you are making_

_please include a reference to an existing issue, if applicable_

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,8 @@
     "ts-node": "^3.0.4",
     "typescript": "^2.3.2",
     "uglify-js": "^3.0.7",
-    "vinyl-fs": "^2.4.4"
+    "vinyl-fs": "^2.4.4",
+    "node-sass-import": "https://github.com/bokzor/node-sass-import"
```
Comment (dherges) [inline package.json]: Isn't there an official version that is released to npm registry?

Comment (bokzor) [reply package.json]: I'm waiting for a merge request but the repo doesn't seems to be active anymore.
I'm going to publish a new version on npm if I have no response until Wednesday

Comment (dherges) [reply package.json]: Hm, what is Angular CLI and the webpack loader using then? Have they implemented the custom importing on their own?

Comment (bokzor) [reply package.json]: They are using css-loader, a plugin for webpack.

Comment (dherges) [reply package.json]: Yes. No. They use sass-loader. But who implements the "~" syntax? sass-loader on their own?

https://github.com/webpack-contrib/sass-loader#imports

Comment (bokzor) [reply package.json]: There is nothing implemented in sass-loader to handle the ~.

From sass-loader : 
webpack provides an advanced mechanism to resolve files. The sass-loader uses node-sass' custom importer feature to pass all queries to the webpack resolving engine. Thus you can import your Sass modules from node_modules. Just prepend them with a ~ to tell webpack that this is not a relative import:

Comment (dherges) [reply package.json]: Apart of there is nothing in their package.json

https://github.com/webpack-contrib/sass-loader/blob/master/package.json#L28-L34

What else is this node-sass-importer doing that ng-packagr didn't do before?

Comment (bokzor) [reply package.json]: I suppose that the sass-loader cannot be use in standalone mode. And that web pack is injecting his resolver.

Ng-packgr was only replacing relative path by complete path. Here, It's the content that is injected.

//File1.scss 
@import('~bootstrap/bootstrap.css');
ng-packagr was replacing this by
//File1.scss 
@import('c:/node_module/path/on/my/computer');

Now
//File1.scss 
//Content of bootstrap.css

Comment (dherges) [reply package.json]: Please provide a reproduction of the bug / previous behaviour along with this PR. There are working builds with...

collapse-card.component.scss:
```scss
@import "../../../styles/src/scss/blubbr";
```

collapse-card.component.ts
```ts
@Component({
    selector: 'u-collapse-card',
    templateUrl: './collapse-card.component.html',
    styleUrls: [ './collapse-card.component.scss' ]
})
```

Compiled result:
```ts
@Component({
    selector: 'u-collapse-card',
    template: ` ...    `,
    styles: [`
      /*!
       * Shared Styles for BLUBBR
       *
       */
      @media print {
        *,
        *::before,
        *::after {
          text-shadow: none !important;
          box-shadow: none !important; }
        a,
        a:visited {
          text-decoration: underline; }
        abbr[title]::after {
          content: " (" attr(title) ")"; }
        pre {
          white-space: pre-wrap !important; }
        pre,
        blockquote {
          border: 1px solid #999;
          page-break-inside: avoid; }
        thead {
          display: table-header-group; }
        tr,
        img {
          page-break-inside: avoid; }
        p,
        h2,
        h3 {
          orphans: 3;
          widows: 3; }
        h2,
...
```

**Comment by dherges** [comment]
As there is no more activity in the last few weeks, I'll close it for now!

Please let know if there are still open issues and/or changes!

**Pull request closed without merging**